Source/WebCore/page/Page.cpp

```diff
@@ -2680,6 +2680,16 @@ void Page::setMuted(MediaProducerMutedStateFlags muted)
     });
 }
 
+bool Page::hasVideoElements()
```

Comment:
Nit: maybe something like `blockLayerTreeFreezingForVideo`?

Reply:
I named it `shouldBlockLayerTreeFreezingForVideo`. Thanks for the suggestion.